projectceladon/vendor-intel-utils
# Fix Bluetooth stability issues

1. Upgrade fw for Bluetooth AX210 Update firmware for Intel Bluetooth AX210 (Typhoon peak - TyP2) to REL73111, which fix the below issue:

During power cycle test, BT FW gone to bad state. So, hci0 interface is not detected because of which com.android.bluetooth process crashed, and the entire Bluetooth frameworks crashed. Upgrading the fw to latest to fix the issue.

Tests done:
Power cycle test step:
1. adb root,  adb shell, reboot -p
2. Manually power off, then manually power on
3. Check if ivi starts normally
4. After step3, BT ON success

Tried this power cycle test 50 times, com.android.bluetooth tombstone crash not observed, able to turn BT ON successfully.

2. Below issues are fixed. OAM-112969
OAM-113245
OAM-114183
OAM-114416

Tests done:
1. Connected the pixel with IVI and checked the metadata sync success. Issue Disconnect & Connect from IVI screen. Metadata sync is success.
2. Launch Aux BT demo app and connect headset. Crash is not observed.
3. During stability testing & monkey testing, the javacrash on A2DP state machine is not observed after the fix.

Tracked-On: OAM-114183
Signed-off-by: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
